ray-operator/controllers/ray/common/association_test.go

```diff
@@ -150,6 +150,28 @@ func TestRayClusterHeadlessServiceListOptions(t *testing.T) {
 	}
 }
 
+func TestRayClusterHeadServiceListOptions(t *testing.T) {
+	instance := rayv1.RayCluster{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "raycluster",
+			Namespace: "test-ns",
+		},
+	}
+
+	labels := HeadServiceLabels(instance)
+	delete(labels, utils.KubernetesCreatedByLabelKey)
+	delete(labels, utils.KubernetesApplicationNameLabelKey)
+
+	expected := []client.ListOption{
+		client.InNamespace(instance.Namespace),
+		client.MatchingLabels(labels),
+	}
+	result := RayClusterHeadServiceListOptions(&instance)
+	if !reflect.DeepEqual(result, expected) {
```

Comment:
Make sure

```go
common.HeadServiceLabels(instance)
```

and

```go
RayClusterHeadServiceListOptions(instance)
```

are basically the same except for the `app.kubernetes.io` labels.